cosmwasm/submit-proposal.js

```diff
@@ -178,13 +178,20 @@ const execute = async (client, wallet, config, options) => {
 };
 
 const registerItsChain = async (client, wallet, config, options) => {
+    const itsAbiTranslator = options.itsTranslatorAddress || config.axelar?.contracts?.ItsAbiTranslator?.address;
```

Comment:
```suggestion
    const itsMsgTranslator = options.itsMsgTranslator || config.axelar?.contracts?.ItsAbiTranslator?.address;
```
ABI is just the default value, but it's more generic